app.js

```diff
@@ -1,29 +1,69 @@
 'use strict';
 
+const port = 3000;
+
```

Comment:
Basically, application port will be configurable and the value will be assign to either application level and Environment level. Read http://12factor.net/config , http://himanshu.gilani.info/blog/2012/09/26/bootstraping-a-node-dot-js-app-for-dev-slash-prod-environment/  or many related topic :)